microservices-connector/README.md

```diff
@@ -61,76 +58,70 @@ make clean
 **Build and push your image to the location specified by `CTR_IMG`:**
 
 ```sh
-make docker-build docker-push CTR_IMG=<some-registry>/gmconnector:tag
+make docker.build docker.push CTR_IMG=<some-registry>/gmcmanager:tag
 ```
 
 **NOTE:** This image ought to be published in the personal registry you specified.
 And it is required to have access to pull the image from the working environment.
 Make sure you have the proper permission to the registry if the above commands don’t work.
 
-**Install the CRDs into the cluster:**
+**Install GMC CRD**
 
 ```sh
-make install
+kubectl apply -f config/crd/bases/gmc.opea.io_gmconnectors.yaml
 ```
 
-**Deploy the Manager to the cluster with the image specified by `CTR_IMG`:**
+**Get related manifests for GenAI Components**
 
 ```sh
-make deploy CTR_IMG=<some-registry>/gmconnector:tag
+mkdir -p $(pwd)/config/manifests
+cp $(dirname $(pwd))/manifests/ChatQnA/*.yaml -p $(pwd)/config/manifests/
```

Comment:
@irisdingbj, besides the manifests yaml files, we may need to copy  gmc router yaml file in directory `GenAIInfra/microservices-connector/config/gmcrouter/`.